### PR TITLE
Fix macOS CI test suite

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Fix macOS CI, see more details here:
https://github.com/actions/virtual-environments/issues/1811#issuecomment-708480190

### CODE REVIEW

**Review by DavidMansolino — APPROVED** [review]
Looks good, please wait for https://github.com/cyberbotics/webots/pull/2367/checks?check_run_id=1285957497 before merging.

**Comment by lukicdarkoo** [comment]
I requested review too early :/ Let's wait for CI to pass

**Comment by lukicdarkoo** [comment]
> Looks good, please wait for https://github.com/cyberbotics/webots/pull/2367/checks?check_run_id=1285957497 before merging.

Exactly :)

**Pull request merged**
